spec/README.md

```diff
@@ -219,83 +219,74 @@ Example (only showing one lower case diacritic):
                     "base": "u",
                     "decompose": "ue"
                 },
-                "equivalents": {
-                    "unicode": "\u00fc",
-                    "html_decimal": "&#252;",
-                    "html_hex": "&#xfc;",
-                    "encoded_uri": "%C3%BC",
-                    "html_entity": "&uuml;",
-                    "extras": {
-                        "unicode": "u\u0308",
+                "equivalents": [
+                    {
+                        "raw": "ü"
+                        "unicode": "\u00fc",
+                        "html_decimal": "&#252;",
+                        "html_hex": "&#xfc;",
+                        "encoded_uri": "%C3%BC",
+                        "html_entity": "&uuml;"
+                    },
+                    {
+                        "raw": "u¨",
```

Comment:
Is this `ü` character correctly? The dots that normally appear above the u appear beside of the u for me.